core/behavior/baseline.py

```
@@ -0,0 +1,149 @@
+
+    def __init__(self, min_messages: int = 50):
+        self.min_messages = min_messages
+        self._global = _ScopeStats()
+        self._scopes: dict[str, _ScopeStats] = defaultdict(_ScopeStats)
+
+    # ---- 喂入消息 ----
+
+    def feed(self, text: str, scope: str = "global", has_sticker: bool = False) -> None:
+        stripped = text.strip()
+        if not stripped and not has_sticker:
+            return  # 纯贴图消息（无文本）也要统计
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)